client/src/components/Upload.vue

```diff
@@ -341,10 +363,12 @@ export default {
             <v-row>
               <v-col cols="auto">
                 <v-checkbox
-                  v-model="pendingUpload.createSubFolders"
+                  :input-value="pendingUpload.createSubFolders"
                   label="Create Subfolders"
+                  disabled
+                  hint="Enabled when many videos are being uploaded"
+                  persistent-hint
```

Comment:
Should we even display a unchecked disabled user input to the user? (If they are uploading an image sequence)  I know that reading it lets you know that you can upload multiple video files at once if you happen to be uploading image sequences.  Either way is fine, just a question.

Reply:
I sorta like it.